frontends/web/src/routes/device/bitbox02/checkbackup.tsx

```diff
@@ -46,66 +46,70 @@ class Check extends Component<Props, State> {
   };
 
   private checkBackup = async () => {
+    const { backups, deviceID, t } = this.props;
+    this.setState({ message: t('backup.check.confirmTitle') });
     try {
-      const backupID = await bitbox02API.checkBackup(this.props.deviceID, true);
-      const foundBackup = this.props.backups.find((backup: Backup) => backup.id === backupID);
+      const backupID = await bitbox02API.checkBackup(deviceID, true);
+      const foundBackup = backups.find((backup: Backup) => backup.id === backupID);
       if (!foundBackup) {
-        alertUser(this.props.t('unknownError', { errorMessage: 'Not found' }));
+        alertUser(t('unknownError', { errorMessage: 'Not found' }));
         return;
       }
       this.setState({
-        foundBackup,
         activeDialog: true,
-        message: this.props.t('backup.check.success'),
```

Comment:
this was wrong, it did set the success title too early. moved to the end after `await bitbox02API.checkBackup(deviceID, false);`